src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -8,6 +8,9 @@
 # pylint: disable=line-too-long
 
 helps['ad sp create-for-rbac'] = """
+            parameters:
+                - name: --cert-name
```

Comment:
We should rename to `--keyvault-cert`, or even `--kv-cert`; otherwise with the existing `--cert`, it is hard to understand when read existing commands or the code itself